src/PageLayout/index.tsx

```diff
@@ -72,13 +72,13 @@ export const PageBody = styled.div<PageBodyProps>`
     grid-template-columns: 1fr;
   }
 
-  & ${PageContent} {
+  & PageContent! {
```

Comment:
component selectors weren't working, changed to this syntax. good find @ciaranschutte